apps/rehabstod/src/pages/CurrentSickLeaves/components/TableBodyRows.test.tsx

```diff
@@ -136,7 +136,7 @@ it('Should render all sickleave columns', async () => {
   )
 
   expect(await screen.findAllByRole('row')).toHaveLength(10)
-  expect(screen.getAllByRole('row')[0].children).toHaveLength(14)
+  expect(screen.getAllByRole('row')[0].children).toHaveLength(15)
```

Comment:
We could get the number of columns from `Object.keys(SickLeaveColumn).length` the same way we do in `PatientTableBody`. But this is not super important, just something I noticed.

Reply:
15 is not the length so I will leave it as it is. I think it will be more confusing if we start adding subtractions in several places